### ISSUE TITLE
Need a hook_config_info() in i18n_select so config file is owned by that module

### ISSUE THREAD

**Issue by herbdool** [backdrop-contrib/i18n#126]
A client can't sync their config because i18n_select.settings isn't owned by any module. Something like the following should fix it:

```
/**
 * Implements hook_config_info().
 */
function i18n_select_config_info() {
  $prefixes['i18n_select.settings'] = array(
    'label' => t('Multilingual select settings'),
    'group' => t('Configuration'),
  );
  return $prefixes;
}
```

**Comment by indigoxela** [comment]
@herbdool many thanks for finding and suggesting the solution! :pray: 

Yes, the missing hook_config_info is just an oversight. Will add it. :+1:

**Comment by indigoxela** [comment]
Fix available in #127

**Comment by herbdool** [comment]
PR works for me, thanks @indigoxela
